### PR TITLE
MGDOBR-293: database setup for multiple shards

### PR DESCRIPTION
**Pull request by r00ta**
[MGDOBR-293](https://issues.redhat.com/browse/MGDOBR-293) 

Please make sure that your PR meets the following requirements:

- [x] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config)
- [x] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [x] All new functionality is tested
- [x] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Annotate your PR with the label `safe to test` in order to run the end to end CI pipeline

### CODE REVIEW

File: manager/src/main/resources/application.properties [hunk 1]
```diff
@@ -19,6 +19,7 @@ quarkus.flyway.migrate-at-start=true
 quarkus.flyway.validate-on-migrate=true
 quarkus.flyway.ignore-missing-migrations=true
 quarkus.flyway.create-schemas=true
+quarkus.flyway.placeholders.shard-id=${SHARD_ID}
```
Comment (danielezonca) [inline manager/src/main/resources/application.properties:22]: Don't we need to update `kustomize` scripts to set this value (both `minikube` and `prod`)?

Comment (r00ta) [reply manager/src/main/resources/application.properties:22]: We have already the `SHARD_ID` there. It was used for the shard api authentication

File: manager/src/main/resources/application.properties [hunk 2]
```diff
@@ -47,6 +48,7 @@ event-bridge.kafka.topic-prefix=ob-
 %dev.quarkus.datasource.password=event-bridge
 %dev.quarkus.hibernate-orm.log.sql=true
 %dev.quarkus.hibernate-orm.log.bind-parameters=true
+%dev.quarkus.flyway.placeholders.shard-id=150b23e9-0b34-45b7-a91a-5ec388d03a1d
```
[the remark this answers is not in the document]
Comment (r00ta) [reply manager/src/main/resources/application.properties:51]: Currently the `shard-id` we are using for `dev` is specified in the keycloak realm and it's the same for everybody

**Review by danielezonca — APPROVED** [review]

**Review by robpblake — APPROVED** [review]

**Pull request merged**